App/Course App/Course App/Views/SwiftUI/HorizontalScrollView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  HorizontalScrollView.swift
+//  Course App
+//
+//  Created by Work on 22.05.2024.
+//
+import os
+import SwiftUI
+
+struct HorizontalScrollView: View {
+    private enum UIConstant {
+        static let radius: CGFloat = 10
+    }
+
+    let logger = Logger()
+    let data: [Joke]
+    
+    var body: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
```

Comment:
Horizontal scroll view should be implemented in UIKit. This solution doesn't recycle memory. In case we have big images or videos after scrolling for a bit the whole app will grab huge amount of memory

Reply:
I would from UX perspective to have paging on place so always end with scrolling on the middle of the screen

Reply:
I found it below 👍

Reply:
i understand